companies/4care-gmbh.json

```diff
@@ -8,9 +8,10 @@
     ],
     "name": "4Care GmbH",
     "address": "Wissenschaftspark\nFraunhoferstr. 17\n24118 Kiel\nDeutschland",
-    "phone": "+49 431 881188",
+    "phone": "+49 431 881188a",
     "fax": "+49 431 881166",
-    "email": "datenschutz@4care.de",
+    "pgp-fingerprint": "",
+    "email": "datenschutz@4care.de.",
```

Comment:
🚫 **[recordlint]** <[pass-schema](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record did not pass schema validation: “The value at /email must be a valid idn-email string but it was not.”

Raw error:
```json
{
    "path": "$.email",
    "pointer": "/email",
    "message": "The value at /email must be a valid idn-email string but it was not.",
    "data": "datenschutz@4care.de."
}
```